src/stcal/jump/jump.py

```diff
@@ -34,7 +34,8 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
                  edge_size=25, extend_snr_threshold=1.2, extend_min_area=90,
                  extend_inner_radius=1, extend_outer_radius=2.6,
                  extend_ellipse_expand_ratio=1.2, grps_masked_after_shower=5,
-                 max_extended_radius=200):
+                 max_extended_radius=200, minimum_groups=3,
+                 minimum_sigclip_groups=100, only_use_ints=True):
```

Comment:
I assume all these new params are explained in doc updates over in the jwst package?

Reply:
Also, since I haven't looked at the jwst PR yet, I'm not sure what `only_use_ints` represents, but at first read the use of `ints` makes the programmer in me think of "integers". Is that what it is, or does it have to do with "integrations"?  If the latter, maybe the param name should try to reflect that in a more obvious way, e.g. something like `only_use_integs`?